docker-compose.yml

```diff
@@ -17,3 +17,4 @@ web:
     - db
   environment:
     - PYTHONUNBUFFERED=1
+    - SERVER_NAME=docker.local:5000
```

Comment:
This one is tricky. I don't have a `docker.local`. I have `docker`. Someone running native Docker would have `localhost`. Does it need to be set here?